packages/zendesk-adapter/src/filters/help_center_locale.ts

```diff
@@ -84,7 +84,7 @@ const filterCreator: FilterCreator = ({ config, client }) => ({
         localeName,
         localeType,
         { id: locale, default: locale === defaultLocale },
-        [ZENDESK, RECORDS_PATH, LOCALE_TYPE_NAME, pathNaclCase(locale)],
+        [ZENDESK, RECORDS_PATH, LOCALE_TYPE_NAME, pathNaclCase(naclCase(locale))],
```

Comment:
nit: use `localeName` instead?
```suggestion
        [ZENDESK, RECORDS_PATH, LOCALE_TYPE_NAME, localeName],
```